ui/library/src/utils/scroll.ts

```diff
@@ -168,3 +169,26 @@ export function getPagePropertiesInPixels(): PageProperties {
 
   return pageProperties;
 }
+
+export function calculateTargetPosition({
```

Comment:
This isn't needed yet, right?

Reply:
aaah yes i missed that one. Will remove it